.github/workflows/build-wsl.yaml

```diff
@@ -167,6 +167,35 @@ jobs:
       - name: Build Bundle
         working-directory: ${{ env.workDir }}
         run:  msbuild .\DistroLauncher.sln /t:Build /m /nr:false /p:Configuration=Release /p:AppxBundle=Always /p:AppxBundlePlatforms="${{ env.AppxBundlePlatforms }}" /p:UapAppxPackageBuildMode=${{ env.UapAppxPackageBuildMode }} -verbosity:normal
+      - name: Collect the PDB files
+        working-directory: ${{ env.workDir }}
+        shell: bash
+        run: |
+          set -eu
+
+          # Launcher PDBs
+
+          for arch in "x64" "ARM64"; do
+            collectTo="debug-database-${{ matrix.AppID }}/launcher/$arch/"
+            mkdir -p "$collectTo"
+
+            findDir="ARM64/Release/"
+            if [ $arch == "x64" ]; then
+              findDir="ARM64/Release/DistroLauncher-Appx/x64/"
```

Comment:
Can we double check if this path correct?

Reply:
One can search in the build-wsl workflow run logs for the compiler invocation with the `/PDB:`. That surprised me at first, the path is different from when I build the appx from the IDE.

Reply:
This is right. 2022-08-03T10:38:37.9059925Z          C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\link.exe /ERRORREPORT:QUEUE /OUT:"C:\Temp\builddir\ARM64\Release\DistroLauncher-Appx\x64\launcher.exe" /INCREMENTAL /ILK:"x64\Release\launcher.ilk" /NOLOGO onecore.lib /MANIFEST /MANIFESTUAC:"level='asInvoker' uiAccess='false'" /manifest:embed /DEBUG /PDB:"**C:\Temp\builddir\ARM64\Release\DistroLauncher-Appx\x64\launcher.pdb**" /SUBSYSTEM:CONSOLE /OPT:REF /OPT:ICF /LTCG:incremental /LTCGOUT:"x64\Release\launcher.iobj" /TLBID:1 /DYNAMICBASE /NXCOMPAT /IMPLIB:"C:\Temp\builddir\ARM64\Release\DistroLauncher-Appx\x64\launcher.lib" /MACHINE:X64 x64\Release\DistroLauncher.res